CONTRIBUTING.md

```diff
@@ -83,6 +83,8 @@ issues and/or Redmine tickets (if applicable).
     `SQL/New_patches/` directory.
     4. For SQL patches that are _optional_ (e.g. those that perform some 
             cleanup), place them in `SQL/Cleanup_patches/`.
+    5. The files for our test database ("Raisinbread") must be updated.
```

Comment:
Actually, maybe modify the sentence a bit to say:
"The files for our test database (Raisinbread) must be updated if they were modified. Detailed instructions..."

Reply:
If you look that's what line 78 says. This is a list under that.

Reply:
actually, line 78 mention only database table changes, not changes done to the actual files.